### ISSUE TITLE
NVDA is not announcing the shortcut key information present for the "Instructions" control. :A11y_Visual Studio Code Copilot Extensions_Add Context_Screen Reader

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#269075]
### GitHub Tags:
#A11yTCS; #A11ySev2; #DesktopApp; #Win11; #Win32; #A11yMAS; #SH-Visual Studio Code Copilot Extensions-Win32-Sept25;#Visual Studio Code Client;#WCAG1.3.1;#AILimited; #Screen Reader; #NVDA; #Info and Relationships; #Element:List;

### Environment Details:
Application Name: Visual Studio Code Copilot Extensions
Visual studio code Version: 1.104.2 (user setup)
Microsoft Windows 11 Enterprise 24H2 Build 26100.6584

### Repro Steps:

1. Turn on NVDA.
2. Open Visual studio Code.
3. Login to GitHub copilot chat.
4. Tab till github copilot chat section.
5. Tab till "Add context" button and press enter.
6. Search for file and add context to your request panel will get open.
7. Tab till "Instructions" option from the list.
8. Tab till copilot chat input field.
9. Observe that NVDA is announcing the shortcut key information present for the "Instructions" list option or not.

### Actual Result:
NVDA is not announcing the shortcut key information present for the "Instructions" option.

### Expected Result:
When the "Instructions" control receives keyboard focus, NVDA should announce the associated shortcut key information "Press Ctrl+Alt+/ to open instructions") 

### User Impact:
Users relying on Screen reader will miss critical interaction cues if shortcut key information is not announced

### Attachment:

https://github.com/user-attachments/assets/265e0982-2d12-4c1c-857e-0136f1afe602

**Comment by meganrogge** [comment]
@TylerLeonhardt is this a known quickpick bug or something one can set for a qp item?

**Comment by connor4312** [comment]
This repro's for me, a Windows/NVDA-specific thing?

https://github.com/user-attachments/assets/e65e6148-f2ff-41cf-9795-5417d9e1c623

**Comment by meganrogge** [comment]
The testers have said what we need to do is surface this in the keyboard shorcuts  @justschen

**Comment by meganrogge** [comment]
We discussed this offline and since this keyboard shortcut doesn't work on Windows, the fact that it's not announced to screen readers is good. We'll remove it visually, thus closing this.
